Add info about new PR checks to readme. (#1255)
trojsten/web: README.md

[![codecov](https://codecov.io/gh/trojsten/web/branch/master/graph/badge.svg?token=t4kSkwFccG)](https://codecov.io/gh/trojsten/web)
![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)

## Getting started

This project uses [pre-commit](https://pre-commit.com/). 
You can install it using `pip install pre-commit` and it's also included in [the project requirements](https://github.com/trojsten/web/blob/master/requirements.txt).

Activate it with 
```bash
pre-commit install --allow-missing-config
```

> ### Running localy

Reviewer: ```suggestion
### Running locally
```

> Please follow the instalation manual here: https://github.com/trojsten/web/wiki/Návod-na-inštaláciu

Reviewer: ```suggestion
Please follow the installation manual here: https://github.com/trojsten/web/wiki/Návod-na-inštaláciu
```

> ### Commiting code

Reviewer: ```suggestion
### Committing code
```

> Please send a pullrequest with descriptive title and all necessary information in the description. Preferably all in english.

Reviewer: ```suggestion
Please send a pull request with a descriptive title and all necessary information in the description. Preferably all in English.
```

> The pullrequest needs to pass automatic checks and have at least one approval from the project maintainters.

Reviewer: ```suggestion
The pull request needs to pass automatic checks and have at least one approval from the project maintainers.
```

> We currently check following things:

Reviewer: ```suggestion
We currently check the following things:
```

- code style

>   - the code must follow [black](https://github.com/python/black) codestyle. The easiest way to achieve this is automatically format with the `black` tool. This is also enforced by the pre-commit check. Note that black is currently not included in the project requirements, because it requires Python 3.6+.

Reviewer: ```suggestion
  - the code must follow [black](https://github.com/python/black) codestyle. The easiest way to achieve this is to automatically format the code with the `black` tool. This is also enforced by the pre-commit check. Note that black is currently not included in the project requirements, because it requires Python 3.6+.
```

  - imports must be sorted alphabetically (within import categories). You can use [`isort`](https://github.com/timothycrosley/isort) to automatically sort imports the correct way. This is also enforced by the pre-commit check.
  - the code must also pass [`flake8`](http://flake8.pycqa.org/en/latest/) lint check.
- tests
  - the PR must pass all tests

>   - the PR should also have a sufficent code coverage.

Reviewer: ```suggestion
  - the PR should also have sufficient code coverage.
```
